UnitTests/SqlPreparation/SqlPreparationFromExpressionVisitorTest.cs

```diff
@@ -331,19 +331,18 @@ public void VisitQuerySourceReferenceExpression ()
       var fakeWhereExpression = Expression.Constant (true);
 
       _stageMock
-          .Expect (mock => mock.PrepareWhereExpression (Arg<Expression>.Matches (e => e is BinaryExpression), Arg.Is (_context)))
-          .WhenCalled (
-              mi =>
-              SqlExpressionTreeComparer.CheckAreEqualTrees (
-                  Expression.Equal (groupJoinClause.JoinClause.OuterKeySelector, groupJoinClause.JoinClause.InnerKeySelector),
-                  (Expression) mi.Arguments[0]))
-          .Return (fakeWhereExpression);
-      _stageMock.Replay();
+          .Setup (mock => mock.PrepareWhereExpression (It.Is<Expression> (e => e is BinaryExpression), _context))
+          .Callback (
+              (Expression expression, ISqlPreparationContext _1) =>
```

Comment:
why do we use _1 here isntead of _ ? 
(entire fire)

Reply:
It's the notation our junior chose when multiple parameters were unused in lambdas (since discards are illegal in signatures he chose `_1`, `,2` etc)

I tried to remove these occurrences but apparently missed some (entire files)